app/main.py

```diff
@@ -1,3 +1,61 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        if isinstance(other, int) or isinstance(other, float):
+            return Distance(self.km + other)
```

Comment:
```suggestion
        return Distance(self.km + other)
```

Reply:
Also if you want to check two or more types you can use `isinstance(other, (float, int))`
But in this case it's not needed

Reply:
Fix it everywhere

Reply:
fixed everything